pkg/constants/constants.go

```diff
@@ -31,6 +31,7 @@ const (
 	EphemeralNodeStorage         = "20Gi"
 	EphemeralClusterName         = "ocne-ephemeral"
 	EphemeralClusterPreserve     = true
+	EphemeralClusterVersion      = "1.30"
```

Comment:
could this value be dynamically determined, so we don't need to remember to update this constant when new k8s versions are added?  There is a table of supported k8s versions in pkg/catalog/versions/k8s.go.  Maybe a function could determine latest version from this table contents.

Reply:
looking into it now

Reply:
This is certainly possible, we could also change the default k8sVersion for all clusters to come from this. Or we can instead remove EphemeralClusterVersion and simply use the same default k8sVersion as all clusters https://github.com/oracle-cne/ocne/blob/9870d04ea23e61888924e589414c50c27be21f89/pkg/constants/constants.go#L48

Reply:
I changed this so that it uses the same k8s version as the default cluster start